include/TimeLineWidget.h

```diff
@@ -123,12 +119,9 @@ class TimeLineWidget : public QWidget, public JournallingObject
 	inline QColor const & getActiveLoopInnerColor() const { return m_activeLoopInnerColor; }
 	inline void setActiveLoopInnerColor(QColor const & activeLoopInnerColor) { m_activeLoopInnerColor = activeLoopInnerColor; }
 
-<<<<<<< HEAD
 	inline QColor const & getActiveLoopTextColor() const { return m_activeLoopTextColor; }
 	inline void setActiveLoopTextColor(QColor const & activeLoopTextColor) { m_activeLoopTextColor = activeLoopTextColor; }
 
-=======
->>>>>>> refs/remotes/origin/master
```

Comment:
```diff
-=======
->>>>>>> refs/remotes/origin/master
```

This is what a merge gone wrong looks like. You should not merge anything. At least not for what you're trying to achieve. When you want to update a branch you `rebase`